curve25519-dalek/src/edwards.rs

```diff
@@ -112,7 +111,7 @@ use {
     subtle::CtOption,
 };
 
-#[cfg(feature = "group")]
+#[cfg(any(test, feature = "rand_core"))]
```

Comment:
This matches how we use `rand_core` in [`scalar.rs`](https://github.com/dalek-cryptography/curve25519-dalek/blob/e3b5328202f4fae269daa88553bea7b4d3f86346/curve25519-dalek/src/scalar.rs#L574)